### ISSUE TITLE
ts-evt is ts-events 2.0

### ISSUE THREAD

**Issue by garronej** [rogierschouten/ts-events#18]
Hey guys,
A long time ago I forked ``ts-event`` to do some improvement in this great library.
Initially, I was planning to submit a PR that could be merged but I ended up doing too many breaking changes and reimplementing everything from scratch.
I kept the core API design that I think is excellent but I scrapped out ``AsyncEvent`` and ``QueuedEvent`` as I don't think they are very useful abstractions and a simple wrapper around ``SyncEvent`` can reproduce their behavior. 
Long story short ``ts-evt`` is now it's own project. I mentioned in the docs that it was heavily inspired by ``ts-events`.  
@episage, @rogierschouten I'd be happy to add you as contributors if you are interested.

**Comment by garronej** [comment]
https://github.com/garronej/ts-evt

**Comment by rogierschouten** [comment]
@garronej looks good!  I had half a mind myself for such a rewrite.  Please do

**Comment by garronej** [comment]
Thanks for being such a good sport 🍻

**Comment by garronej** [comment]
And if it is not too much to ask could you update the readme to mention ts-evt's existence.
ts-evt has been carefully tested and is ready for production now. It will even be available on Deno soon. 
The two lib have almost the same name and API, a lot of users will get confused between the two if things stays like this.
Again I am willing to give you credit.

**Comment by rogierschouten** [comment]
Done.

**Comment by garronej** [comment]
Great thank you very much.
 I will mention you more clearly in the README in the next version I push.
And if you want to appear in contributors just submit a PR :)

Good day

**Comment by garronej** [comment]
And sorry last thing, could you publish a minor version to NPM so the update on the readme appears there as well?
